shoop/core/models/_contacts.py

```diff
@@ -91,6 +113,12 @@ def __init__(self, *args, **kwargs):
             kwargs.setdefault("tax_group", self.default_tax_group_getter())
         super(Contact, self).__init__(*args, **kwargs)
 
+    def get_price_display_options(self):
+        group_with_options = self.groups.with_price_display_options().first()
```

Comment:
I think first() is not correct here. IMO if contact has multiple contact groups set with pricing options, then default group should determine the options.